pkg/statistics/handle/autoanalyze/refresher/refresher_test.go

```diff
@@ -351,6 +366,9 @@ func TestAnalyzeHighestPriorityTablesWithFailedAnalysis(t *testing.T) {
 	insertFailedJobForPartitionWithStartTime(tk, "test", "t1", "p0", startTime)
 
 	r.AnalyzeHighestPriorityTables()
+	require.Eventually(t, func() bool {
+		return len(r.GetRunningJobs()) == 0
+	}, 10*time.Second, 100*time.Millisecond)
```

Comment:
10 seconds is excessive; most of these are set to 3 or 5 seconds

Reply:
Updated.

Reply:
I updated them to use `WaitAutoAnalyzeFinishedForTest`. We can avoid it at all.